meta-facebook/yv35-rf/src/platform/plat_gpio.c

```diff
@@ -16,59 +16,47 @@ const char *const gpio_name[] = {
 GPIO_CFG plat_gpio_cfg[] = {
 	//  chip,      number,   is_init, is_latch, direction,    status,     property,    int_type,              int_cb
 	/** Group A: 00-07 **/
-	{ CHIP_GPIO, 0, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_LOW, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
-	{ CHIP_GPIO, 1, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_LOW, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
-	{ CHIP_GPIO, 2, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_HIGH, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
+	{ CHIP_GPIO, 0, ENABLE, ENABLE, GPIO_OUTPUT, GPIO_LOW, PUSH_PULL, GPIO_INT_DISABLE, NULL },
+	{ CHIP_GPIO, 1, ENABLE, ENABLE, GPIO_OUTPUT, GPIO_LOW, PUSH_PULL, GPIO_INT_DISABLE, NULL },
+	{ CHIP_GPIO, 2, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_LOW, PUSH_PULL, GPIO_INT_DISABLE, NULL },
```

Comment:
Both should be an open drain output because this signal is pulled high to 1.8V. 
PP will also work, but OD is better.

Reply:
Done.